src/components/AudioCell.vue

```diff
@@ -117,15 +117,15 @@ const emit =
   defineEmits<{
     (
       e: "focusCell",
-      payload: { audioKey: AudioKey; focusTextField: boolean }
+      payload: { audioKey: AudioKey; focusTarget: "textField" | "root" }
     ): void;
   }>();
 
 defineExpose({
   audioKey: computed(() => props.audioKey),
-  focusCell: ({ focusTextField }: { focusTextField: boolean }) => {
-    if (focusTextField) {
-      textfield.value?.focus();
+  focusCell: ({ focusTarget }: { focusTarget: "textField" | "root" }) => {
```

Comment:
どういう時にどっちにするか分からなくなりそうなので、ドキュメントで案内してあげるのはどうでしょう。
こんな感じ？
```suggestion
  /**
   * フォーカスする。
   * 大体の操作ではtextFieldにフォーカスすればOK。
   * 複数選択などではrootフォーカスする。
   */
  focusCell: ({ focusTarget }: { focusTarget: "textField" | "root" }) => {
```

デフォルト値を`textField`にすれば2行目のドキュメントは消せると思います。